packages/atomic/template/tsconfig.json

```diff
@@ -12,7 +12,9 @@
     "noUnusedParameters": true,
     "jsx": "react",
     "jsxFactory": "h",
-    "types": ["@coveo/atomic"]
+    "paths": {
+      "@coveo/headless": ["./node_modules/@coveo/atomic/headless"]
+    }
```

Comment:
Prevents any mismatch, and getting "double suggestions" for both `coveo/atomic/headless` & `coveo/headless`

Reply:
If I'm understanding this properly, this won't be necessary on the next major release of atomic, because the typing will go away, correct?

Reply:
In the next major, we plan not to have `@coveo/atomic/headless`. 

It will still be bundled in the CDN for this use case (which was why I bundled Headless with Atomic initially) https://docs.coveo.com/en/atomic/latest/usage/headless-through-atomic/ 
Looking back, having it has an NPM subpackage & a dependency at the same time was a mistake.

I can remove the paths config for now, the typing issue is not as bad now that we have the same Atomic/Headless & Headless NPM versions.